### ISSUE TITLE
Change to second sentence of summary.

### ISSUE THREAD

**Issue by terracoda** [phetsims/friction#93]
@zepumph , I have updated the strings for the opening summary.
https://docs.google.com/document/d/1v7GM_dNZs86jcRn2X03D2t7HFDb4dQ1h4ayDGXTu63U/edit#heading=h.3kz4xdtel5zt

In addition, I just wanted to note that there is no need for the break aways to described in teh summary as this is better as an alert.
Using the A11y View, I noticed this:
> In zoomed-in view where books meet, atoms break away, and surface temperature is very hot.

Let me know if you concerns or questions about the new dynamic summary.

**Comment by zepumph** [comment]
Re comment in doc:
>I flipped the sentence around. If this is a huge pain to manage, I can reconsider the order.

It is so very easy, see how simple the above commit is. 

next for this issue I will:
- [ ] update the new text for the jiggling atoms clause
- [ ] remove the break away described in this sentence.

**Comment by zepumph** [comment]
Here is some scratch strings was partial work on this, I think I'm going to implement the logic from decreasing temp alerts in #94 before heading over here.

```js
    zoomedInViewOfWhereBooksMeetPattern: {
      value: 'In zoomed-in view of where books meet, {{atomsTemperatureClause}}.'
    },

    atomsTempPatternAndJigglePattern: {
      value: 'atoms {{temperatureClause}}, and {{jigglingClause}}'
    },

    atomsJiggleSurfaceTempPattern: {
      value: 'atoms jiggle a tiny bit, {{thermometerTemperature}}'
    },
```

**Comment by terracoda** [comment]
@zepumph, the change to the second sentence looks great, except in two cases, I hear and extra "jiggle" which is kind of cute, but not exactly what we want ;-)

The two cases can be easily seen in the A11y View:
- in initial cool state: "[...]surface temperature is cool, and atoms jiggle jiggle a tiny bit."
- in very hot state: "[...]surface temperature is very hot, and atoms jiggle jiggle a lot."

Just remove the extra "jiggle" and this issue is good to close.

**Comment by terracoda** [comment]
@zepumph, you can fix the "jiggle jiggle" in issue #104 if that makes more sense. The scenarios for the changing/transitioning states are working fine, so maybe we should close this issue and deal with the rest in issue #104.
